src/main/java/org/candlepin/subscriptions/tally/admin/InternalTallyResource.java

```diff
@@ -111,4 +124,157 @@ public void purgeTallySnapshots() {
       log.warn("A tally snapshots purge job is already running.");
     }
   }
+
+  /**
+   * Clear tallies, hosts, and events for a given org ID. Enabled via ENABLE_ACCOUNT_RESET
+   * environment variable. Intended only for non-prod environments.
+   *
+   * @param orgId Red Hat orgId
+   */
+  @Override
+  public String deleteDataAssociatedWithOrg(String orgId) {
```

Comment:
@kartikshahc we decided to return JSON objects previously instead of just strings for these internal endpoints like in: https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/swatch-system-conduit/src/main/java/org/candlepin/subscriptions/conduit/admin/InternalOrganizationSyncResource.java

https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/swatch-system-conduit/src/main/spec/internal-organizations-sync-api-spec.yaml

Reply:
I thought to implement as a json response but, was confused to what are the practices here. Thanks for providing the decision. I will make the changes.